frontend/server/src/DAO/SubmissionFeedback.php

```diff
@@ -64,24 +67,26 @@ public static function getFeedbackBySubmission(
                 )
             )
         );
+        $clause = 'AND sf.range_bytes_start IS NULL';
+        $params = [$guid];
```

Comment:
```suggestion
        $params = [$guid];
        $clause = 'AND sf.range_bytes_start IS NULL';
```

así es más claro que la cláusula no es la que contiene el guid.